src/components/application_manager/src/smart_object_keys.cc

```diff
@@ -454,6 +454,10 @@ const char* const rect = "rect";
 const char* const x = "x";
 const char* const y = "y";
 const char* const preferred_fps = "preferredFPS";
+
+// OnResetTimeout
+const char* const request_id = "requestID";
+const char* const timeout_period_for_reset = "resetPeriod";
```

Comment:
```suggestion
const char* const reset_period = "resetPeriod";
```
or
```suggestion
const char* const timeout_reset_period = "resetPeriod";
```
might be a better name

Reply:
@ShobhitAd renamed in https://github.com/smartdevicelink/sdl_core/pull/3726/commits/57e4accc52957086f89470059f8fa07d9ee5dd7c